doc/ADMIN_fr.md

```diff
@@ -0,0 +1,14 @@
+## Configuration
+
+* Tout d'abord, vous aurez besoin d'un Linky et d'un [compte Enedis](https://mon-compte-client.enedis.fr/)
+* Dans votre espace Enedis, rendez-vous dans l'onglet ["Gérer l'accès à mes données"](https://mon-compte-particulier.enedis.fr/donnees/), et permettez à Enedis de collecter et stocker votre consommation horaire
+* Après cela, installez æneria sur votre instance Yunohost, visitez-la et suivez simplement les instructions
+* Ça y est, attendez maintenant un peu pour voir les données apparaître !
+
+## Venant de Pilea
+
+æneria est le nouveau nom de Pilea. Ce nouveau nom est accompagné d'une toute nouvelle version.
+
+Cette nouvelle version a introduit quelques changements majeurs dans la structure des données : la migration depuis Pilea ne peut pas se faire automatiquement.
+
+Vous pouvez facilement transférer vos données de Pilea vers æneria en suivant [cette documentation](https://docs.aeneria.com/fr/latest/administrateur/pilea_migration.html).
```

Comment:
I think you can completely delete this part. Nobody should still be on Pilea now.